rportfolio/src/components/nav/Nav.jsx

```diff
@@ -0,0 +1,79 @@
+import "./nav.css"
+
+import React from 'react'
+
+function Nav() {
+  return (
+    <div className="navSection">
+        <div className="NavLeft">
+          <div className="logo">Akutsang</div>
+        </div>
+        <div className="navRight">
+            <div className="home">Home</div>
+            <div className="about">About</div>
+            <div className="skills">Skills</div>
+            <div className="contact">Contact</div>
```

Comment:
Your nav bar items are supposed to be links correct? Or you don't intend to navigate users to the respective pages/sections. If yes, why and why not routes?